Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -605,6 +643,12 @@ def main():
                     default=None,
                     help='Specify a platform specific data (HEX, DWORD) for customization')
 
+    ap.add_argument('-tpm',
```

Comment:
As other arguments, better to use '-t', '--tpm' '-t' if no one is using it.

Reply:
Yeah, Will use "-t" option.